.travis.yml

```diff
@@ -0,0 +1,27 @@
+sudo: required
+
+language: bash
+
+services:
+  - docker
+
+install:
+  - docker build -t quay.io/wikiwatershed/rwd:${TRAVIS_COMMIT:0:7} .
+
+script:
+  - docker run -d --name rwd quay.io/wikiwatershed/rwd:${TRAVIS_COMMIT:0:7}
```

Comment:
If no `CMD` or `ENTRYPOINT` is provided, I think it just runs `sh` or `bash` as the default, but I'm not sure.

If we have a way to sanity check the things inside of the image, it would be great to try and do that here. That said, we shouldn't worry about pulling down 2TB on each test run to confirm that everything works.